GenderPayGap.WebUI/Views/AdminFeedback/ViewFeedback.cshtml

```diff
@@ -0,0 +1,266 @@
+﻿@using GenderPayGap.Core
+@using GenderPayGap.Database.Models
+@using GenderPayGap.WebUI.Models.Admin
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model List<GenderPayGap.Database.Models.Feedback>
+
+@{
+    ViewBag.Title = $"View feedback - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+{
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = "View feedback"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+@{
+    object FeedbackRows(List<Feedback> feedbackRows, string buttonClasses)
+    {
+        Func<object, object> currentStateHtml = @<text>
+            @foreach (Feedback feedback in feedbackRows)
+            {
+                <tr class="govuk-table__row">
+                    <td class="govuk-table__cell" rowspan="7" style="border-bottom: 2px solid #b1b4b6;">

```

Comment:
It doesn't feel great having colours here. Is it worth setting up a stylesheet?

Reply:
Good idea. In the end I didn't even need the colours.